buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -1027,14 +1028,14 @@ class BeamModulePlugin implements Plugin<Project> {
       project.apply plugin: 'net.ltgt.errorprone'
 
       project.dependencies {
-        errorprone("com.google.errorprone:error_prone_core:2.3.1")
+        errorprone("com.google.errorprone:error_prone_core:$errorprone_version")
         errorprone("jp.skypencil.errorprone.slf4j:errorprone-slf4j:0.1.2")
         // At least JDk 9 compiler is required, however JDK 8 still can be used but with additional errorproneJavac
         // configuration. For more details please see https://github.com/tbroyer/gradle-errorprone-plugin#jdk-8-support
         errorproneJavac("com.google.errorprone:javac:9+181-r4173-1")
       }
 
-      project.configurations.errorprone { resolutionStrategy.force 'com.google.errorprone:error_prone_core:2.3.1' }
+      project.configurations.errorprone { resolutionStrategy.force 'com.google.errorprone:error_prone_core:2.3.4' }
```

Comment:
Shouldn't this use `$errorprone_version` too?

Reply:
I saw the single quote syntax being used here and decided to leave it alone. To use the variable, I'll need switch to double-quotes. Let me try that.

Reply:
Thanks @ibzib for taking care of this before I got to it.